test/tests/mqtrigger_v2/kafka/hellokafka.js

```diff
@@ -0,0 +1,15 @@
+module.exports = async function (context) {
```

Comment:
nitpick: rename directories from mqtrigger_v2 to mqtrigger_kind_keda

Reply:
Done